executor.go

```diff
@@ -1152,16 +1152,59 @@ func (e *executor) executeRowsShard(_ context.Context, index string, fieldName s
 		return nil, ErrFieldNotFound
 	}
 
-	// Rows query does not currently support a `time` field that has
-	// `noStandardView: true`.
-	// TODO https://github.com/pilosa/pilosa/issues/1783
-	if f.Type() == FieldTypeTime && f.options.NoStandardView {
-		return nil, errors.New("Rows() query on time field with no standard view is not currently supported")
-	}
+	// rowIDs is the result set.
+	var rowIDs RowIDs
+
+	// views contains the list of views to inspect (and merge)
+	// in order to represent `Rows` for the field.
+	var views []string = []string{viewStandard}
+
+	// Handle `time` fields.
+	if f.Type() == FieldTypeTime {
+		var err error
+
+		// Parse "from" time, if set.
+		var fromTime time.Time
+		if _, ok := c.Args["from"]; ok {
+			switch v := c.Args["from"].(type) {
+			case string:
+				if fromTime, err = time.Parse(TimeFormat, v); err != nil {
+					return nil, errors.New("cannot parse Row() 'from' time")
+				}
+			case int64:
+				fromTime = time.Unix(v, 0).UTC()
+			default:
+				return nil, errors.New("Row() 'from' arg must be a timestamp")
+			}
+		}
 
-	frag := e.Holder.fragment(index, fieldName, viewStandard, shard)
-	if frag == nil {
-		return make(RowIDs, 0), nil
+		// Parse "to" time, if set.
+		var toTime time.Time
+		if _, ok := c.Args["to"]; ok {
```

Comment:
Does it make sense to limit `toTime` to today + 1 day like for the `Row` query?

Reply:
.. of course as long as `to` is not explicitly specified.

Reply:
sounds like we'd need to rewrite things such that `viewsByTimeRange` can take the actual available views into account rather than naively generating all possible views in the range for the quantum. 

I'm fine with *requiring* both `from` and `to` if that's too much work right now.

I'm not fine with defaulting to now+1 day which I think is arbitrary/surprising behavior that will cause bugs for people.

Reply:
I believe most of the users would use a timestamp for past or current time, and 1 day is simply something to account for time zone differences. Did we have any use case where we had to use a future timestamp?

In any case, the best solution is using already existing views if we can, instead of iterating on all possible views.

Reply:
ok, the latest commit first determines the min/max possible time range values based on the available views. then, the default time range uses those min/max values.